<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>panel_InfosJrs</class>
 <widget class="QWidget" name="panel_InfosJrs">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>315</width>
    <height>570</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>315</width>
    <height>570</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>315</width>
    <height>570</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(229, 229, 229);</string>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>16</x>
     <y>0</y>
     <width>300</width>
     <height>570</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>300</width>
     <height>570</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>300</width>
     <height>570</height>
    </size>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(193, 193, 193);</string>
   </property>
   <property name="tabPosition">
    <enum>QTabWidget::West</enum>
   </property>
   <property name="tabShape">
    <enum>QTabWidget::Triangular</enum>
   </property>
   <property name="currentIndex">
    <number>-1</number>
   </property>
   <property name="usesScrollButtons">
    <bool>false</bool>
   </property>
   <property name="tabsClosable">
    <bool>true</bool>
   </property>
   <property name="movable">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_MasquerPanel">
   <property name="geometry">
    <rect>
     <x>1</x>
     <y>1</y>
     <width>14</width>
     <height>30</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>14</width>
     <height>30</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>14</width>
     <height>30</height>
    </size>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="../../mesRessources.qrc">
     <normaloff>:/images/divers/masquer</normaloff>:/images/divers/masquer</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>64</width>
     <height>64</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_AfficherPanel">
   <property name="geometry">
    <rect>
     <x>1</x>
     <y>31</y>
     <width>14</width>
     <height>30</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>14</width>
     <height>30</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>14</width>
     <height>30</height>
    </size>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="../../mesRessources.qrc">
     <normaloff>:/images/divers/afficher</normaloff>:/images/divers/afficher</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>64</width>
     <height>64</height>
    </size>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="../../mesRessources.qrc"/>
 </resources>
 <connections/>
</ui>
